R/info.R

```diff
@@ -1,3 +1,18 @@
+# get version of rust-polars version from Cargo.lock at package build time
+# from polars 0.31.1 this can be migrated to rust side see
+# https://github.com/pola-rs/polars/pull/9660
+RUST_POLARS_VERSION = (\() {
+  Cargo.lock = readLines("./src/rust/Cargo.lock")
```

Comment:
I don't think this will be working user side.

Reply:
I think it would work. In what scenario? If all files are deleted after compilation?

Reply:
It happens at package build time.

Reply:
just tried and I can confirm it works:

``` r
library(polars)
list.files(".")
#> [1] "ok-coqui_reprex.R"        "ok-coqui_reprex.spin.R"  
#> [3] "ok-coqui_reprex.spin.Rmd"
pl$polars_info()
#> r-polars package version : 0.7.0.9000
#> rust-polars crate version: 0.30.0
#> 
#> Features:          
#> simd FALSE
```
(also works in another project directory)

Reply:
Ok, thanks.